tests/data/test_count.py

```diff
@@ -1,11 +1,14 @@
 from functools import reduce
+from typing import Any
 
 import pytest
 
 import ixmp4
+from ixmp4.data.backend import Backend
 
 
-def deepgetattr(obj, attr):
+# TODO How to type hint this to return Repos with .list() and .count()?
+def deepgetattr(obj: Backend, attr: str) -> Any:
```

Comment:
This type hint is not great, but I don't know how to make it better.

Reply:
Well, use `object`, but that means you will always have to check the type of the return value afterwards.

Reply:
We can do the same kind of type check with `Any`, which I have done now. This brought to light some missing kwargs for `count()`, which is great :)